packages/rancher-istio-1.9/charts/values.yaml

```diff
@@ -80,6 +78,3 @@ kiali:
 tracing:
   enabled: false
   contextPath: "/jaeger"
-  jaeger:
-    repository: rancher/mirrored-jaegertracing-all-in-one
-    tag: 1.20.0
```

Comment:
does this also exist in the jaeger chart?

Reply:
https://github.com/rancher/charts/blob/1c33d37907da925fd25760fb6b36952a19d8899a/packages/rancher-tracing/charts/values.yaml#L24 << yep, exact same situation as kiali.